src/main/java/yapper/Yapper.java

```diff
@@ -0,0 +1,84 @@
+package yapper;
+
+import java.util.Scanner;
+
+import yapper.instructions.InstructionHandler;
+import yapper.io.InputFileHandler;
+import yapper.io.StringStorage;
+import yapper.tasks.TaskHandler;
+
+/**
+ * Main class for Yapper.
+ *
+ * <p>
+ * The Yapper class implements a simple command-line chatbot.
+ * It interacts with the user through a loop that:
+ * processes user input, handles tasks, and manages program flow.
+ * <p/>
+ *
+ */
+public class Yapper {
+
+    /**
+     * Contains the main Chatbot Loop
+     *
+     * <p>
+     * Starts the main chatbot loop, allowing the user to interact
+     * with the chatbot until they decide to exit.
+     * <p/>
+     *
+     * @param taskHandler the TaskHandler instance that manages tasks
+     */
+    public static void startYappin(TaskHandler taskHandler) {
```

Comment:
while the name is "cool", for someone maintaining the code, what does `startYappin` mean? use a name that represents the operation